src/SamplesApp/SamplesApp.Droid/MainActivity.cs

```diff
@@ -22,13 +22,16 @@ namespace SamplesApp.Droid
 			ConfigurationChanges = global::Uno.UI.ActivityHelper.AllConfigChanges,
 			WindowSoftInputMode = SoftInput.AdjustPan | SoftInput.StateHidden
 		)]
+	[IntentFilter(
+		new[] { Android.Content.Intent.ActionMain },
+		Categories = new[] { Android.Content.Intent.CategoryLauncher, Android.Content.Intent.CategoryLeanbackLauncher })]
```

Comment:
Will users have to manually add this in their apps? Should it be documented and/or included in templates?

Reply:
Note: This is included for Xamarin but not net6.0+?